piton/piton_settings.bash

```diff
@@ -129,7 +129,10 @@ then
 NEWPATH=$NEWPATH:$CC_BIN
 fi
 
-export PATH="$NEWPATH:$PATH"
+export PATH="$PATH:$NEWPATH"
```

Comment:
Inb4 this breaks for someone else

Reply:
added the old version for that poor soul to try